src/layer/S1GRDAWSEULayer.ts

```diff
@@ -135,4 +135,22 @@ export class S1GRDAWSEULayer extends AbstractSentinelHubV3Layer {
       hasMore: response.data.hasMore,
     };
   }
+
+  public async findDates(
+    bbox: BBox,
+    fromTime: Date,
+    toTime: Date,
+    orbitDirection?: OrbitDirection,
+  ): Promise<Date[]> {
+    const findDatesDatasetParameters: S1GRDFindTilesDatasetParameters = {
+      type: this.dataset.shProcessingApiDatasourceAbbreviation,
```

Comment:
Are you certain this is correct for all datasets? Currently, we specify multiple string on the dataset, each of them being used in a different scenario. In this sense, `shProcessingApiDatasourceAbbreviation` is used only when interacting with Processing API. I think we shouldn't use it here, or if we do, we should find a better name for it (so that it is clear it can safely be used with this endpoint).

EDIT: to clarify, I think we should add another identifier to the dataset, which can be used here...

Reply:
Thanks, did some testing and found that we were lucky as the `type` is the same as `shProcessingApiDatasourceAbbreviation ` for S1GRD but not necessarily for others.
Also found out that we use `shProcessingApiDatasourceAbbreviation ` in `findTiles` as well (here in `S1GRDAWSEULayer` at least).

Will fix.

Reply:
Should be fixed now.